cmd/serve.go

```diff
@@ -464,6 +438,36 @@ func (c *ServeCommand) Command(serverService ServerServiceInterface, monitorServ
 		Run: func(cmd *cobra.Command, _ []string) {
 			ctx := cmd.Context()
 
+			// Setup the Admin DB connection pool
+			dbcpOptions := di.DBConnectionPoolOptions{DatabaseURL: globalOptions.DatabaseURL, MonitorService: monitorService}
+			adminDBConnectionPool, err := di.NewAdminDBConnectionPool(ctx, dbcpOptions)
+			if err != nil {
+				log.Ctx(ctx).Fatalf("error getting Admin DB connection pool: %v", err)
+			}
+			defer func() {
+				di.DeleteInstanceByValue(ctx, adminDBConnectionPool)
```

Comment:
Maybe `di.DeleteInstanceByValue(ctx, adminDBConnectionPool)` would be a better name ? Given that we're deleting or closing, depending on the instance type.

Reply:
Renamed it to `DeleteAndCloseInstanceByValue` in https://github.com/stellar/stellar-disbursement-platform-backend/pull/207/commits/f7b0beca00188fef27559e4f0b7d9cba3252d7e8